docs/concepts/governance.md

```diff
@@ -228,6 +235,23 @@ Another parameter is
 
 <!--### Changing models [WIP]  -->
 
+### Propose a successor market
+A successor market is a market that will carry on after the original market, or parent, that it is based on has settled - though a parent and successor market can be active simultaneously. Proposing a new successor market that follows from an existing market allows liquidity providers to the option to keep their [equity-like share](./liquidity/rewards-penalties.md#how-liquidity-fees-are-split) on the new market, even when the original market expires. Creating an entirely new market with no parent doesn't offer the same benefit.
+
+Each market can have only one active successor.
+
+In terms of the proposal format, there are only two differences between a succesor market proposal and that for a regular market.  
+* Parent market ID: Required to define the proposal as for a successor market
+* Insurance pool percentage: Required percentage of the parent market's insurance pool, up to 100%, can be earmarked for transfer to the successor market. It is submitted as a number between and including 0 and 1, which represents the factor for the percentage.
+
+The following details will need to match between parent and successor:
+* Settlement asset
+* Quote: Determined from the data source
```

Comment:
Quote?

Reply:
@TomMcL I'll take suggestions, as I don't think "quote asset" will necessarily always be true. Quote name?

Reply:
I don't think that has to be the same at all? A rolling 3m future would be settling on a very different oracle which doesn't inherently have a tie to the previous one